angular-workspace/projects/ni/nimble-angular/src/directives/anchor-button/tests/nimble-anchor-button-router-link-with-href.directive.spec.ts

```diff
@@ -3,6 +3,7 @@ import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testin
 import { Router } from '@angular/router';
```

Comment:
Discussed how the parameterize test helper could be useful in skyline as well and is maybe not good to expose via nimble-components or nimble-angular packages (it's a generic jasmine helper not tied to our components).

Directions discussed:
- Create a separate package in nimble, like @ni/jasmine-parameterized (which is similar to this [public package](https://github.com/paucls/jasmine-parameterized/blob/master/package.json) except ours would be in the ni scope)
  - Would need typescript compile and jasmine test set-up
  - Would actually be neat to run it in node and and in browser, should be generic enough to, the typescript build would need to emit dist/esm and dist/cjs
- Copy and paste the utility to the @ni/nimble-angular/internal-utilities entry point so it isn't part of the publically supported api and make the package later in nimble or the js style guide.

Reply:
Moving to draft temporarily until a direction is picked

Reply:
In #1738, I am creating a new @ni/jasmine-parameterized package with these utilities. Once that PR is merged, I plan to update this PR to reference the new location of `parameterizeNamedList`.